### PR TITLE
[LM-10799] Resolved peer testing bugs

### PR DESCRIPTION
**Pull request by prabha-likeminds**
## Description

**Features**

**Bugs**
1. Swipe on multiple media posts not working (navigating to post detail screen instead)
2. Comment input box keyboard doesn’t open when we navigate to post detail screen by clicking comments count
3. No callback to go to Like Screen
4. No callback for back button for all screens
5. No callback for delete post and comment
6. No callback for report post and comment
7. No callback for overflow menu for post and comment
8. No callback for create post button in universal feed
9. No callback for Post button in create post
10. No callback for user item in likes screen

**What kind of change does this PR introduce?** (check at least one)

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [x] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [x] Label added
- [x] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [x] Description about the Feature/ Bugs added

### CODE REVIEW

File: example/SampleApp/sampleApp/feedScreen/createScreen.tsx [hunk 1]
```diff
@@ -14,10 +14,21 @@ const CreateScreen = ({navigation}) => {
     handleGallery(type);
     console.log('after gallery handle');
   };
+  const customHandlecreatePost = (allAttachment,formattedLinkAttachments, postContentText) => {
```
Comment (jaixchaudhary) [inline example/SampleApp/sampleApp/feedScreen/createScreen.tsx:17]: name is not completely camel case

File: likeminds-feed-reactnative-integration/context/createPostContext.tsx [hunk 2]
```diff
@@ -228,6 +231,25 @@ export const CreatePostContextProvider = ({
     });
   };
 
+  const onPostClick = async (allMedia:Array<LMAttachmentUI>, linkData: Array<LMAttachmentUI>, content: string) => {
```
Comment (jaixchaudhary) [inline likeminds-feed-reactnative-integration/context/createPostContext.tsx:234]: add comments for this method

File: likeminds-feed-reactnative-integration/context/createPostContext.tsx [hunk 3]
```diff
@@ -548,6 +570,10 @@ export const CreatePostContextProvider = ({
     }
   };
 
+  const handleScreenBackPress = () => {
```
Comment (jaixchaudhary) [inline likeminds-feed-reactnative-integration/context/createPostContext.tsx:573]: add comments

File: likeminds-feed-reactnative-integration/context/postLikesListContext.tsx [hunk 4]
```diff
@@ -105,12 +106,17 @@ export const PostLikesListContextProvider = ({
     }
   }, [route.params]);
 
+  const handleScreenBackPress = () => {
```
Comment (jaixchaudhary) [inline likeminds-feed-reactnative-integration/context/postLikesListContext.tsx:109]: add comments

File: likeminds-feed-reactnative-integration/context/universalFeedContext.tsx [hunk 5]
```diff
@@ -160,14 +171,34 @@ export const UniversalFeedContextProvider = ({
...
+        showToastMessage({
+          isToast: true,
+          message: POST_UPLOADED,
+        })
+      );
     }
     return addPostResponse;
   };
 
+  const newPostButtonClick = () => {
```
Comment (jaixchaudhary) [inline likeminds-feed-reactnative-integration/context/universalFeedContext.tsx:184]: add comments

**Review by jaixchaudhary — APPROVED** [review]

**Review by ishaan1607 — APPROVED** [review]

**Pull request merged**